onnxruntime/core/session/inference_session.h

```diff
@@ -270,6 +278,8 @@ class InferenceSession {
   std::string EndProfiling();
 
  protected:
```

Comment:
>protected [](start = 1, length = 9)

Is there any function should be set as private?

Reply:
kept a minimum set of protected ones in new iteration.